### DISCUSSION TITLE
Octoapp Companion for K1 Max not installing

### DISCUSSION THREAD

**Discussion by gschmidl** [Guilouz/Creality-Helper-Script-Wiki#560] [Issues]
On the latest firmware and version of the script, Octoapp Companion does not install:

```
Running in K1 and K1 Max OS mode
Checking required system packages are installed...
Requirement already satisfied: virtualenv in /usr/lib/python3.8/site-packages (15.1.0)
System package install complete.
Checking Python Virtual Environment For OctoApp...
Updating PIP if needed... (this can take a few seconds or so)
Could not fetch URL https://pypi.python.org/simple/pip/: There was a problem confirming the ssl certificate: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1108) - skipping
Requirement already up-to-date: pip in /usr/data/octoapp-env/lib/python3.8/site-packages
Installing or updating required python libs...
  Could not find a version that satisfies the requirement websocket_client<1.6.99,>=1.6.0 (from -r /usr/data/octoapp/requirements.txt (line 10)) (from versions: )
No matching distribution found for websocket_client<1.6.99,>=1.6.0 (from -r /usr/data/octoapp/requirements.txt (line 10))
```

**Comment by Guilouz** [comment]
@crysxd Can you check this please.

@gschmidl My script use installer from Octoapp, please post issue directly on repo : https://github.com/crysxd/OctoApp-Plugin

**Reply by Guilouz** [reply]
This have been fixed for Octoapp Companion

**Reply by crysxd** [reply]
This is the same issue: 

https://github.com/crysxd/OctoApp-Plugin/issues/73
https://github.com/QuinnDamerell/OctoPrint-OctoEverywhere/issues/60

**Comment by psiess** [comment]
I'm having the same issue when trying to install Octoeverywhere.

Running in K1 and K1 Max OS mode
Checking required system packages are installed...
System package install complete.
Checking Python Virtual Environment For OctoEverywhere...
No virtual environment found, creating one now.
Already using interpreter /usr/bin/python3
Using base prefix '/usr'
/usr/lib/python3.8/site-packages/virtualenv.py:1039: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
New python executable in /usr/data/octoeverywhere-env/bin/python3
Also creating executable in /usr/data/octoeverywhere-env/bin/python
Installing setuptools, pip, wheel...done.
Updating PIP if needed... (this can take a few seconds or so)
Could not fetch URL https://pypi.python.org/simple/pip/: There was a problem confirming the ssl certificate: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1108) - skipping
Requirement already up-to-date: pip in /usr/data/octoeverywhere-env/lib/python3.8/site-packages
Installing or updating required python libs...
  Could not find a version that satisfies the requirement websocket_client<1.7.99,>=1.6.0 (from -r /usr/data/octoeverywhere/requirements.txt (line 10)) (from versions: )
No matching distribution found for websocket_client<1.7.99,>=1.6.0 (from -r /usr/data/octoeverywhere/requirements.txt (line 10))

**Reply by psiess** [reply]
Firmware V1.3.3.26

**Reply by Guilouz** [reply]
@psiess Please report this on OctoEverywhere github: https://github.com/QuinnDamerell/OctoPrint-OctoEverywhere

**Reply by psiess** [reply]
Done.  Thanks @Guilouz

**Reply by crysxd** [reply]
https://github.com/QuinnDamerell/OctoPrint-OctoEverywhere/issues/60

**Comment by Guilouz** [comment]
It's now fixed.
